### PR TITLE
Add test database to CI config

### PR DESCRIPTION
**Pull request by RobertWSaunders**
## Proposed Change

This adds a test database to our Circle CI config so that our server tests can pass!

### How did you do this?

Using the `circleci/postgres:9.6.5-alpine-ram` image that CircleCI provides!

### Why are you choosing this approach?

Make our tests actually work!

### Any open questions you want to ask code reviewers?

Nope!

### List of changes:

  - Update our `config.yml`

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new tests for my core changes (where applicable).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,25 +6,37 @@ jobs:
...
+          command: |
+            npm run bootstrap
+            sudo apt-get install postgresql-client
+      - run:
+          name: Create test database
+          command: |
+            psql -U postgres -h localhost -p 5432 -c "CREATE DATABASE \"qhacks-test-database\";"
       - run:
-          name: Build MJML Email Templates
+          name: Build MJML emails
```
Comment (joeytepp) [inline .circleci/config.yml]: This step may no longer be necessary with the new server!

Comment (RobertWSaunders) [reply .circleci/config.yml]: Totally correct, I forget why we even needed this last time...? Can you remind me?

Comment (RobertWSaunders) [reply .circleci/config.yml]: I removed it!

Comment (joeytepp) [reply .circleci/config.yml]: The mock-server relied on having the MJML templates to start, pretty sure it was from `require`-ing the emails package, no longer the case though!

Comment (RobertWSaunders) [reply .circleci/config.yml]: Ah nice, maybe my email refactor helped that. It was quite a mess, not sure why that was ever a requirement haha.

**Pull request merged**